tests/nnstreamer_filter_armnn/unittest_filter_armnn.cc

```diff
@@ -362,7 +362,9 @@ TEST (nnstreamer_filter_armnn, invoke_01)
   void *data = NULL;
   gchar *model_file, *data_file;
   const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
-  GstTensorMemory input_uint8, output, input;
+  gchar *input_uint8_data = NULL;
+  gsize input_uint8_size = 0;
```

Comment:
Why did you seperate data and size from GstTensorMemory?

Reply:
input_uint8 does not used for invoke callback. It just reads data file and fills input.